apps/web/package.json

```diff
@@ -22,6 +22,7 @@
 		"@heroicons/react": "^2.0.12",
 		"@jitsi/react-sdk": "^1.3.0",
 		"@jitsu/jitsu-react": "^1.3.0",
+		"@nextui-org/react": "^2.2.9",
```

Comment:
@Cedric921 I love nextui lib, but I wonder how much more effort will be needed to reuse existed libs we already reference (e.g. https://github.com/shadcn-ui/ui, etc)?